index.html

```diff
@@ -1,309 +1,500 @@
+            <li><a href="#about-section" class="nav-link">Contact</a></li>
+          </ul>
+          <button id="open-menu-btn"><i class="uil uil-bars"></i></button>
+          <button id="close-menu-btn"><i class="uil uil-multiply"></i></button>
         </div>
+      </div>
+    </div>
+    <section id="wrapper">
+      <!--Header Section -->
+      <!-- <section id="header">
+            <nav class="nav">
+                <h3 class="logo">My Logo</h3>
+                <div class="menu-icon">
+                    <i class="uil uil-bars"></i>
+                </div>
+            </nav>
+        </section> -->
 
-    </header>
-
-        <!-- Headline Section -->
+      <!-- Header Desktop -->
+      <section id="header-desktop">
+        <nav class="nav">
+          <h3 class="logo">My Logo</h3>
+          <ul class="menu-icon">
+            <li><a href="">Portfolio</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Contact</a></li>
+          </ul>
+        </nav>
+      </section>
 
-    <section id="headline">
-        <h1 class="intro">I’m Esther <br> Glad to see you!</h1>
-        <p class="text">
-            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      <!-- Headline Section -->
+      <section id="headline">
+        <h1 class="intro">
+          I’m Esther <br />
+          Glad to see you!
+        </h1>
+        <p class="headline-para">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
         </p>
 
-        <h4 class="connect">LET’S CONNECT</h4>
-
-            <ul class="headline-icons">
-                <li><a href=""><i class="uil uil-twitter"></i></a></li>
-                <li><a href=""><i class="uil uil-linkedin-alt"></i></a></li>
-                <li><a href=""><i class="uil uil-medium-m"></i></a></li>
-                <li><a href=""><i class="uil uil-github"></i></a></li>
-                <li><a href=""><i class='fa fa-angellist'></i></a></li>   
-            </ul>
+        <h4>LET’S CONNECT</h4>
+        <ul class="headline-icons">
+          <li>
+            <a href=""><i class="uil uil-twitter"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="uil uil-linkedin-alt"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="uil uil-medium-m"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="uil uil-github"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa fa-angellist"></i></a>
+          </li>
+        </ul>
+      </section>
     </section>
 
     <!-- Works Section -->
     <section id="cards">
```

Comment:
The content of this section needs to be dynamically generated.

<img width="1063" alt="Screen Shot 2022-06-24 at 8 25 28" src="https://user-images.githubusercontent.com/82426682/175545292-05caa65c-1801-4ab7-8780-09da0b04c3c1.png">

Please do so.